qutebrowser/browser/commands.py

```diff
@@ -968,6 +968,8 @@ def tab_focus(self, index: typing.Union[str, int]=None, count=None):
                    Negative indices count from the end, such that -1 is the
                    last tab.
             count: The tab index to focus, starting with 1.
+            zero_count: The tab index to focus, with 0. The special value
+                   0 value focuses the last focused tab.
```

Comment:
This should be in the description for `count` now, `zero_count` doesn't exist as argument to `tab_focus` anymore.